lib/Controller/Attribute/ResponseHeader.php

```diff
@@ -28,6 +26,66 @@
  *
  */
 #[\Attribute(\Attribute::IS_REPEATABLE | \Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
-final class ResponseHeader extends BaseResponseHeader
+final class ResponseHeader
 {
+    protected string $key;
+
+    protected string|array $values;
+
+    protected bool $replace = false;
+
+    /**
+     * @param string|array|null $key
+     * @param string|array $values
+     * @param bool $replace
+     */
+    public function __construct($key = null, $values = '', $replace = false)
```

Comment:
We should prefere type hints instead of php docs
```suggestion
    public function __construct(string|array|null $key = null, string|array $values = '', bool $replace = false)
```